### DISCUSSION TITLE
Thoughts for a Change Log?

### DISCUSSION THREAD

**Discussion by gregfenton** [mmazzarolo/react-native-modal-datetime-picker#760] [General]
I am upgrading from v13.1.2 to v18.  Some features of v13.1.2 are no longer there (e.g. headerTextIOS).  I found the PR for the change so now understand what to do.  But I wonder if there are thoughts of a very light-weight CHANGELOG that just bulletpoints the add/drop/fix of a feature, one entry per release?

If such a thing exists already, apologies in advance.

**Comment by mmazzarolo** [comment]
Hola! Check out the releases section: https://github.com/mmazzarolo/react-native-modal-datetime-picker/releases

**Reply by gregfenton** [reply]
Million thanks.  Don't know why I didn't think about checking there. . .
